### PR TITLE
CORE-20139: Refactorings and LB 4.27.0 upgrade

### PR DESCRIPTION
**Pull request by malachyb**
The dependabot upgrade of Liquibase had some deprecated functions that need to be removed, so this PR changes to the new framework liquibase uses.

Linked to https://github.com/corda/corda-runtime-os/pull/5892 as it could not be reopened

### CODE REVIEW

**Comment by malachyb** [comment]
build e2e

File: libs/db/db-admin/build.gradle [hunk 1]
```diff
@@ -12,4 +12,11 @@ dependencies {
 
     compileOnly "org.osgi:osgi.annotation"
     api "javax.persistence:javax.persistence-api"
+
+    implementation libs.liquibase
+    constraints {
```
Comment (vkolomeyko) [inline libs/db/db-admin/build.gradle]: I do not think we need this `constraints` as per: https://mvnrepository.com/artifact/org.liquibase/liquibase-core/4.27.0 LB is now using snakeYaml v2.2.

In fact we can grep across the whole codebase and remove all the `constraints` with "required until liquibase-core updates it's internal version of snakeYaml, currently using 1.33 which has CVE-2022-1471".

**Review by vkolomeyko — COMMENTED** [review]
Overall looks reasonable. One comment posted.

**Comment by vkolomeyko** [comment]
build e2e

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Review by kyriathar — APPROVED** [review]
OK

**Pull request merged**